### DISCUSSION TITLE
Get the value of the nodes, by the connector.

### DISCUSSION THREAD

**Discussion by fedor5792** [freeplane/freeplane#603] [Help] [answered]
Hello! How can I get the value of the nodes, by the connector. I will need to summarize the values.
[map.zip](https://github.com/freeplane/freeplane/files/9451835/map.zip)

**Comment by macmarrum** [comment, marked as answer]
You are on a good track.

```groovy
// For outgoing connectors
node.connectorsOut.collect { it.target.to.num0 }.sum()

// For incoming connectors
node.connectorsIn.collect { it.source.to.num0 }.sum()
```
[fedor5792~New map+.zip](https://github.com/freeplane/freeplane/files/9453866/fedor5792.New.map%2B.zip)

**Reply by fedor5792** [reply]
Thanks so much!

**Comment by euu2021** [comment]
Interesting use of connectors!
